kong/clustering/init.lua

```diff
@@ -53,22 +64,31 @@ function _M:init_cp_worker(plugins_list)
 
   events.init()
 
+  self.json_handler:init_worker(plugins_list)
   self.wrpc_handler:init_worker(plugins_list)
 end
 
+
 function _M:init_dp_worker(plugins_list)
   local start_dp = function(premature)
     if premature then
       return
     end
 
-    self.child = require("kong.clustering.wrpc_data_plane").new(self.conf, self.cert, self.cert_key)
+    if kong.configuration.wrpc_hybrid_protocol then
```

Comment:
We should either make this happen at the init phase or move the require to file scope.

Reply:
`data_plane` will run in only one worker process, so it need not to be inited for all workers.